ssh-legion

```diff
@@ -143,7 +177,7 @@ EOF
     FNAME="${FNAME}+${SALTED_MACHINE_ID}"; fi'
     "echo -n ${COMPRESSED_INFO@Q} | base64 -d | gunzip > ~/connections/\${FNAME}"
     'rm -f ~/connections/"${FNAME}+disconnected"'
-    'sleep infinity&' # Run sleep forever in background until killed by trap
+    "sleep ${CONNECTION_TIME@Q} &" # Run sleep forever in background until killed by trap
```

Comment:
In bash, the `$[variable@Q}` means:

> `${parameter@operator}`
>   The expansion is either a transformation of the value of parameter or information about parameter itself, depending on the value of operator. Each operator is a single letter:
> ...
>
> > **Q**
> >  The expansion is a string that is the value of parameter quoted in a format that can be reused as input.

(from https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html)

Essentially, if you do `@Q` on a string like `'My string with spaces and "quotes"'`, it will output `\'My\ string\ with\ spaces\ and\ \"quotes\"\'`, such that when bash parses it, it turns into the same string.

This is required because after expansion in our client, the `sleep ...` command will be parsed again by bash on the server.